tikv-configuration-file.md

```diff
@@ -12,6 +12,17 @@ The TiKV configuration file supports more options than command-line parameters.
 
 This document only describes the parameters that are not included in command-line parameters. For more details, see [command-line parameter](/command-line-flags-for-tikv-configuration.md).
 
+## Global configuration
+
+### abort-on-panic
+
++ Sets whether to call `abort()` to exit the process when TiKV panics. This option affects whether TiKV allows the system to generate core dump files.
+
+    + If the value of this configuration item is `false`, when TiKV panics, it calls `exit()` to exit the process.
+    + If the value of this configuration item is `true`, when TiKV panics, TiKV calls `abort()` to exit the process. At this time, TiKV allows the system to generate core dump files when exiting. To generate a core dump file, you also need to perform the system configuration related to the core dump (for example, enable the `ulimit -c` and configure the core dump path. Different operating systems have different configuration methods). To avoid the core dump file occupying too much disk space and causing insufficient TiKV disk space, it is recommended to set the core dump generation path to different disk partitions of TiKV data.
```

Comment:
```suggestion
    + If the value of this configuration item is `true`, when TiKV panics, TiKV calls `abort()` to exit the process. At this time, TiKV allows the system to generate core dump files when exiting. To generate a core dump file, you also need to perform the system configuration related to the core dump (for example, setting core dump file size limit via `ulimit -c` command, and/or configure the core dump path. Different operating systems have different related configurations). To avoid the core dump file occupying too much disk space and causing insufficient TiKV disk space, it is recommended to set the core dump generation path a disk partition different to that of TiKV data.
```